src/protect-function-app/README.md

```diff
@@ -0,0 +1,130 @@
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Enable Function App Authentication
+
+1. Open the existing Function App in the Azure portal.
+1. Click on **Authentication** and then **Add identity provider**.
+1. Use these settings in your identity provider configuration.
+
+   | Identity provider setting       | Value for this sample app                                | Notes                                                                            |
+   |--------------------------------:|:---------------------------------------------------------|:---------------------------------------------------------------------------------|
+   | **Identity provider**           | **Microsoft**                                            | Required value for this sample.                                                  |
+   | **App registration type**       | **Provide the details of an existing app registration**  | Required value for this sample.                                                  |
+   | **Application (client) ID**     | `<client-id>`                                            | Required value for this sample. <br/> 'Application (client) ID' of app registration in Azure portal - this value is a GUID               |
+   | **Client secret (recommended)** | _None_                                                   | Suggested value for this sample. <br/> This sample doesn't require this feature. |
+   | **Issuer URL**                  | `https://sts.windows.net/<Tenant ID>/v2.0`               | Required value for this sample. <br/> Update to include 'Tenant ID' of your Azure AD instance - this value is a GUID                     |
+   | **Allowed token audiences**     | `api://<client-id>`                                      | Required value for this sample. <br/> 'Application ID URI' of app registration in Azure portal - this value typically starts with api:// |
+   | **Restrict access**             | **Require authentication**                               | Required value for this sample.                                                  |
+   | **Unauthenticated requests**    | **HTTP 401 Unauthorized: recommended for APIs**          | Suggested value for this sample.                                                 |
+   | **Token store**                 | _Unselected_                                             | Suggested value for this sample. |
+
+   > :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+1. Click **Add**.
```

Comment:
```suggestion
```

Reply:
Removed in Simplification applied in https://github.com/Azure-Samples/ms-identity-docs-code-python/pull/10/commits/852d23cbe8069ceaf40758dec86e08a12e509063